scripts/deploy

```diff
@@ -39,6 +43,10 @@ case "$OUTPUT_FORMAT" in
   env)
     write_env "$OUT/addresses"
     ;;
+  *)
+    copy_template "$TEMPLATE" "$OUTPUT_FORMAT"
+    write_template "$OUTPUT_FORMAT"
```

Comment:
should `write_template` take the output file not the output format?

Reply:
Yep. `OUTPUT_FORMAT` is variable that contain format or path to file.
Think I'll rename it to `DEPLOYMENT_OUTPUT`